src/bot/commands/whois.js

```diff
@@ -1,89 +1,131 @@
+import createLogger from 'another-logger';
+const log = createLogger({label: 'cmd:whois'});
 import {Command} from 'yuuko';
 import config from '../../../config';
 import {parseUser, formatDate} from '../util/discord';
 import {escape} from '../util/formatting';
 
 // function for generating the reddit info
 async function redditLine (userID, guildID, db) {
-	const results = await db.collection('redditAccounts').find({userID, guildID}).toArray();
-	return `__Reddit accounts: **${results.length}**__${
-		results.map(r => `\n- <https://www.reddit.com/u/${encodeURIComponent(r.redditName)}>`).join('')
-	}`;
+	let message = '__Reddit accounts: N/A__';
+
+	try {
+		const results = await db.collection('redditAccounts').find({userID, guildID}).toArray();
+		message = `__Reddit accounts: **${results.length}**__${
+			results.map(r => `\n- <https://www.reddit.com/u/${encodeURIComponent(r.redditName)}>`).join('')
+		}`;
+	} catch (error) {
+		log.error('Couldn\'t Talk to Database', error);
```

Comment:
Let's make these error messages specific to the function they're logged from to make it a bit clearer where the issue lies without having to check the traceback. "Couldn't fetch reddit accounts while constructing whois message" or something similar for each of these.

Reply:
Now the error messages are specific